en/includes/guides/authentication/passwordless-login/add-passwordless-login-with-passkey.md

```diff
@@ -151,3 +152,80 @@ Follow the steps below to use an enrolled passkey to sign in to an application.
+2. In its **Advanced** tab, under **Trusted App Settings**, select **Add as a FIDO trusted app**.
+
+3. Under **Platform Settings**, enter the following platform-specific details.
+    
+    - For an Android app:
+
+        - Provide the package name of the application which takes the reverse domain format (e.g. com.example.myapp)
+
+        - Provide key hashes, which are SHA256 fingerprints of the app's signing certificate.
+    
+    - For an iOS app:
+
+        - Provide the app ID of your application which consists of the team ID and the bundle ID separated by a period (.). (e.g. A1B2C3D4E5.com.domainname.applicationname)
+    
+    !!! note
+
+        {% if product_name == "Asgardeo" %}
+        Validation services of Android and iOS require the application data under **Platform Settings** to be available in the following public endpoints common to all Asgardeo organizations. 
+
+        - For Android - `{{base_url}}/.well-known/assetlinks.json`
+
+        - For iOS - `{{base_url}}/.well-known/apple-app-site-association`
+
+        Alternatively, you may publish to a custom endpoint by setting up [custom domains]({{base_path}}/guides/branding/configure-custom-domains/). Make sure the data are in the format expected by the validation services of [Android](https://developer.android.com/training/app-links/verify-android-applinks#web-assoc){target="_blank"} and [iOS](https://developer.apple.com/documentation/xcode/supporting-associated-domains){target="_blank"} and are publicly hosted in the following locations.
+
+        - For Android - `{custom_domain}/.well-known/assetlinks.json`
+
+        - For iOS - `{custom_domain}/.well-known/apple-app-site-association`
+
+        {% else %}
+        Validation services of Android and iOS require the application data under **Platform Settings** to be available in the following publicly accessible endpoints of your organization domain. 
+
+        - For Android - `{organization_domain}/.well-known/assetlinks.json`
+
+        - For iOS - `{organization_domain}/.well-known/apple-app-site-association`
+
+        {{product_name}} hosts these data in the following endpoints.
+
+        - For Android - `{{base_url}}/.well-known/trusted-apps/android`
+
+        - For iOS - `{{base_url}}/.well-known/trusted-apps/ios`
+
+        Therefore, ensure public requests to `/assetlinks.json` and `/apple-app-site-association` on your domain are mapped to the corresponding local endpoints of {{product_name}}.
+
+        {% endif %}
+
+        Learn more about the validation services of [Android](https://passkeys-auth.com/docs/implementation/flutter/android/){target="_blank"} and [iOS](https://developer.apple.com/documentation/xcode/supporting-associated-domains){target="_blank"}
+
+    !!! warning "Third-Party Data Exposure"
+
+        While not a security concern, it is still important to note that other organizations are able to access the data under **Platform Settings** by accessing the corresponding public endpoint.
```

Comment:
For Asgardeo, should emphasis that custom domain users won't be impacted by this.